yage-gl/src/objects/buffer.rs

```diff
@@ -0,0 +1,66 @@
+use crate::{GL, GlFunctions};
+
+/// Wrapper around an OpenGL array or element array buffer. 
+pub struct Buffer<'a> {
+    gl: &'a GL,
+    /// Target for use in `glBindBuffer`
+    target: u32,
+    vbo: <GL as GlFunctions>::GlBuffer,
```

Comment:
Why vbo? This is just the buffer handle, right? So it can be anything, not necessarily a VBO. So maybe buffer_handle, buffer, or handle?